src/main.tsx

```
@@ -6,11 +6,15 @@ import { createRoot } from 'react-dom/client';
 import { HelmetProvider } from 'react-helmet-async';
 import { ToastContainer } from 'react-toastify';
 
+import { getSentryReactErrorHandlerOptions, initSentry } from '@/shared/config/sentry.ts';
```

ESLint

[error] Replace `·getSentryReactErrorHandlerOptions,·initSentry·}·from·'@/shared/config/sentry.ts'` with `⏎··getSentryReactErrorHandlerOptions,⏎··initSentry,⏎}·from·"@/shared/config/sentry.ts"`

(prettier/prettier)